### DISCUSSION TITLE
Trzymanie danych i oznaczenia

### DISCUSSION THREAD

**Discussion by qaziok** [cvlab-ai/polish_banknotes#33] [Announcements]
## Dane, dane, dane
Dobra - tak jak wcześniej pisałem na Discord zgłosiłem nasz projekt do [v7labs](https://www.v7labs.com/) i z radością mogę ogłosić, że dostaliśmy dostęp do wersji edukacyjnej. Oznacza to, że mamy dostęp do przechowywania 50 tysięcy zdjęć i przyjemnego (wgl mnie) tagowania i oznaczania tych plików. Co więcej, tworzenie zbiorów do trenowania różnych algorytmów i integracja przez bibliotekę z Pythonem.

### A co z OneDrive?
Jeśli ktoś nie czuje potrzeby korzystania z narzędzia przeznaczone stricte do pracy z danymi treningowymi, to nie widzę przeszkód, żeby updatować oba zbiory danych - sam zamierzam się przenieść na v7.

### A więc, czy są wady?
Jedynym problemem, jaki widzę jest limit na konta - w wersji edukacyjnej możemy mieć jedynie 3. Jedno przepada na właściciela (chyba się liczy; składałem wniosek, więc wiadomo, że to ja), dwa pozostałe możemy dzielić między sobą. Proponuję, skoro wszyscy będziemy korzystali z tych danych, żeby olać podział na grupy Android i iOS i po prostu się dzielić dowolnie. Sam swoje też zamierzam udostępnić. 

## Tutorial 
_Przynajmniej tyle na początek (będę go updatował w miarę)_

Logujemy się [tutaj](https://darwin.v7labs.com/login), passy sobie udostępnimy jakoś i witamy w panelu.

W tym [miejscu](https://www.v7labs.com/academy) znajduje się akademia v7, gdzie możecie znaleźć przeróżne poradniki co i jak.

**Comment by StanislawSm** [comment]
Jak powinniśmy robić ze zdjęciami, które są klatkami filmów? Wybrać kilka reprezentatywnych, czy wrzucić wszystkie?
Dodatkowo chciał bym zaznaczyć, że wszystkich zdjęć banknotu 50 zł jest ponad 10 000, jeżeli będzie tak z każdym nominałem to zabraknie nam miejsca.

**Reply by kjedrasz2137** [reply]
Nie wrzucaj wszystkich @StanislawSm , wybierz tylko kilka. Jak będzie około 3-5 'takich samych' zdjęć to nie będzie tragedii, jak się różnią trochę to zostaw. Ja przed usunięciem miałem 17,700 zdjęć a po usunięciu już tylko 5,600. Dobrze by było jakby też Ci zostało około 5,500 - 5,900 zdjęć na koniec, chociaż i tak będziemy rozbudowywać nasz zbiór to może nie ma to tak wielkiego znaczenia teraz. 
Też mi się wydaje, że zabraknie miejsca, średnio w jednej klasie będzie mogło być maksymalnie 50,000/7 = 7142 zdjęć, a już mamy po 6000 zdjęć
